version/version.go

```diff
@@ -16,5 +16,18 @@ limitations under the License.
 package version
 
 var (
-	Version = "v0.0.0"
+	// This variable shows the user any additional changes to files made that aren't on the branch.
+	GitDirty string
+	// This variable shows the commit hash of the repo so they can confirm they are on latest or specific version of the branch.
+	GitHash string
+	// This variable is dependent on what the current release is e.g. if it is v0.2.3 then this variable, outside of releases, will be v0.2.4-dev .
+	Version = "v0.2.4-dev"
 )
+
+func init() {
+	if GitDirty != "" && GitHash != "" {
+		Version += " (" + GitHash + " - " + GitDirty + ")"
```

Comment:
Keep version as it is. 

Do this when printing to stdout from build time variables. 

And use `fmt.Sprintf`, try to avoid string processing operations. It is confusing and more prone to errors.

Reply:
Makes sense, will do!